modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc

```diff
@@ -30,11 +30,23 @@ Please bear in mind:
 * This app requires at minimum ownCloud Version 10.5 and php 7.1.
 ====
 
+== Business Flow for Microsoft 365
```

Comment:
I would not use the wording "business flow" here as this term in this context relates to the enforced authentication via MS (`wopi.business-flow.enabled`).

Reply:
Discussed with @tbsbdr - got the point now. Will change the headline.

Reply:
The headline is fixed now, @tbsbdr pls re-check and approve if ok.